src/App.jsx

```diff
@@ -1,12 +1,34 @@
+import './index.css';
 import './App.css';
 import TextQuotes from './TextQuotes';
+import React, { useState } from 'react';
 
 function App() {
+  const [category, setCategory] = useState('all');
+
+  const handleCategoryChange = (event) => {
+    setCategory(event.target.value);
+  };
+  
+  const handleRefresh = () => {
+    window.location.reload();
+  };
+
   return (
     <> 
-      <TextQuotes/>
+           <h1 className="title">Get Inspired</h1>
+      <select value={category} onChange={handleCategoryChange}>
+        <option value="all">All</option>
+        <option value="motivational">Motivational</option>
+        <option value="inspirational">Inspirational</option>
```

Comment:
Since we unfortunately don't have these different categories right now, I would just give one option for now (plus possibly the "all" option) but still add the code that then selects the relevant content (in this case all of the quotes that we have at the moment).